e2e/tests/ui-driven/src/helpers/context.ts

```diff
@@ -6,6 +6,14 @@ import { log } from "./globalHelpers";
 
 type NewTeam = Parameters<CoreDomainClient["team"]["create"]>[0];
 
+export interface Flow {
```

Comment:
nit: Personally I'd start extracting the types from this file into their own file, it starts making less sense when there are interfaces with different names from the name of the file.

Reply:
I think it is worthwhile having a fuller `Flow` type from `planx-core` but feel it sits outwith this PR, as I'd like to keep this focused on the tests. I'll queue up the change in a separate PR and add it to my list for this sprint